awx/api/views/__init__.py

```diff
@@ -3438,10 +3438,6 @@ def get_queryset(self):
         return getattr(parent, self.relationship).all()
 
     def is_valid_relation(self, parent, sub, created=False):
-        mutex_list = ('success_nodes', 'failure_nodes') if self.relationship == 'always_nodes' else ('always_nodes',)
```

Comment:
:+1:

Reply:
Can confirm, this test is currently failing on the `always_nodes_ui` branch:
```py.test -c config/docker.cfg --base-url='http://localhost:8013' -k 'test_node_triggers_should_be_mutually_exclusive' --disable-pytest-warnings```